jcache/embedded/src/test/java/org/infinispan/jcache/JCacheManagerUriNotHierarchicalTest.java

```diff
@@ -0,0 +1,65 @@
+package org.infinispan.jcache;
+
+import static org.infinispan.jcache.util.JCacheTestingUtil.withCachingProvider;
+import static org.testng.AssertJUnit.assertEquals;
+
+import java.io.IOException;
+import java.net.URI;
+
+import javax.cache.Cache;
+import javax.cache.CacheManager;
+
+import org.infinispan.jcache.embedded.JCacheManager;
+import org.testng.annotations.Test;
+
+/**
+ * Testing case where path to infinispan.xml contain "file:" schema.
+ * <p>
+ * Created by Tomasz Zablocki on 31.07.16.
+ */
+@Test(groups = "unit", testName = "jcache.JCacheManagerUriNotHierarchicalTest")
+public class JCacheManagerUriNotHierarchicalTest {
+
+   public void testJCacheManagerWherePathContainsFileSchema() throws IOException {
+      withCachingProvider(provider -> {
+         //given
+         CacheManager manager = new JCacheManager(
+               URI.create("file:infinispan_uri.xml"),
+               provider.getClass().getClassLoader(),
+               provider,
+               null);
+         Cache fooCache = manager.getCache("foo");
+         String keyName = "sweet";
+         String expected = "chicken";
+         fooCache.put(keyName, "chicken");
+
+         //when
+         String actual = (String) fooCache.get(keyName);
+
+         //then
+         assertEquals(expected, actual);
+      });
+   }
+
+   public void testJCacheManagerWherePathContainsJarFileSchema() throws IOException {
+      withCachingProvider(provider -> {
+         //given
+         CacheManager manager = new JCacheManager(
+               URI.create("jar:file:infinispan_uri.xml"),
+               provider.getClass().getClassLoader(),
+               provider,
+               null);
+         Cache fooCache = manager.getCache("foo");
+         String keyName = "wrong";
```

Comment:
Do we need to obtain a cache? Perhaps creating `JCacheManager` instance would be enough for this test?